.yamato/config/universal_lighting.metafile

```diff
@@ -127,9 +127,10 @@ platforms:
         flavor: b1.xlarge
         model: rtx2080
       editmode:
```

Comment:
Why are all the yml changes also present in the meta file? Not specific to this PR...

Reply:
there are specific changes for this PR - this is the first time we're using a GPU for editmode tests

Reply:
@sophiaaar still what is the purpose of the metafile?

Reply:
does it also need to be set to b1.xlarge?

Reply:
the metafile is what we use to define the rules for the jobs, and then we run a script to generate the ymls. this is so we avoid things like copy-paste errors etc

Reply:
can i make a commit on your branch just to check the ymls have all been generated?